### ISSUE TITLE
Question: DB dump of current data?

### ISSUE THREAD

**Issue by Akurosia** [lmcintyre/ResLogger2#1]
Is there a way to get the current DB dump of all resources, to build a haslist.db for the explorer-fork by goat?

**Comment by lmcintyre** [comment]
Yep, there is a basic database dump feature in the File menu in the plugin. However, you probably actually want the full hash list from crowdsourcing, which is available at https://rl2.perchbird.dev/export.csv.

Both exports will need some stripping to be imported into Explorer.

**Comment by Akurosia** [comment]
ok i played around with it a bit but it seams to be missing elements that are already in the existing hashlist.
Is there a way to provide manual entries to the DB? (e.g. i made a completed list for 060000 but goat did not merged it with a comment to this list here)

**Comment by lmcintyre** [comment]
You can combine the hash lists. The existing hash list in Explorer will just accept the additional entries from any of your exported path lists, and them to it's database.

Eventually this will be a bit more seamless.

**Comment by Akurosia** [comment]
so there is no way to provide missing entries into your DB manually?
would be good to have a single complete source without everyone having to merge them every time
